GetIntoTeachingApiTests/Controllers/SchoolsExperience/CandidatesControllerTests.cs

```diff
@@ -240,12 +240,14 @@ public void AddClassroomExperienceNote_ValidRequest_EnqueuesJobAndRespondsWithNo
                 It.IsAny<EnqueuedState>()));
         }
 
-        private static bool IsRequestPlusId(Candidate requestCandidate, string candidateSentToJobJson, Guid expectedId)
+        private static bool IsRequestPlusIdAndApiGeneratedIdFlag(Candidate requestCandidate, string candidateSentToJobJson, Guid expectedId)
```

Comment:
I re-read this method name a few times 😅 how about something like `MatchesCandidateWithUpfrontId`?